articles/install-guide/csharp.md

```diff
@@ -20,19 +20,19 @@ To write C# programs, download and install the **.NET Core SDK** 2.1 or later fr
 ## Installation ##
 
 To download the latest templates for creating new C# projects and libraries that invoke Q# operations, execute this from a command line:
```

Comment:
This part needs a clarification that you need to do it only if you've installed from command line, if you're doing it in VS you're done.